### ISSUE TITLE
Base-line classif: Use cross_validate with scoring = 'neg_log_loss'

### ISSUE THREAD

**Issue by estevinho** [andreiarog/KaggleCompetitions#18]
https://scikit-learn.org/stable/modules/model_evaluation.html

Why scikit learn version of the score will output negative values: https://stackoverflow.com/questions/26282884/why-is-the-logloss-negative 

Log Loss:
The goal of our machine learning models is to minimize this value. A perfect model would have a log loss of 0. Log loss increases as the predicted probability diverges from the actual label. 

Log loss above 1: https://stackoverflow.com/questions/35013822/log-loss-output-is-greater-than-1

To output probability predictions per observation: https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.cross_val_predict.html#sklearn.model_selection.cross_val_predict
